src/spetlr/cache/CachedLoaderParameters.py

```diff
@@ -41,4 +47,6 @@ def __init__(
         self.rowHash = "rowHash"
         self.loadedTime = "loadedTime"
         self.deletedTime = "deletedTime"
+
         self.do_nothing_if_more_rows_than = do_nothing_if_more_rows_than
+        self.provisional_markup_step = provisional_markup_step or True
```

Comment:
Why "or True"? 
Is it brcause if any set the markup to None or?

Reply:
Oh! I need to remove the or True.